tests/ExampleTest.php

```diff
@@ -1,11 +1,51 @@
 <?php
 
+declare(strict_types=1);
+
 use PHPUnit\Framework\TestCase;
 
 class ExampleTest extends TestCase
 {
-    public function test_it_just_example()
+    private App\Example $example;
+
+    public function setUp(): void
+    {
+        $this->example = new App\Example();
+    }
+
+    public function testHello()
+    {
+        $this->assertSame("hi", $this->example->hello());
+    }
+
+    /**
+     * @dataProvider dataProviderForAddition
```

Comment:
dataProvider in phpunit is very important. For writing efficient tests, you must need to write tests efficiently, one function test with different scenario (input and expected output).
https://docs.phpunit.de/en/9.5/writing-tests-for-phpunit.html#data-providers